__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const app = require('../app');
+const seed = require("../db/seeds/seed");
+const testData = require('../db/data/test-data');
+const db = require("../db/connection")
+
+beforeEach(() => {
+    return seed(testData);
+});
+
+afterAll(() => {
+    return db.end();
+});
+
+describe("GET /api/topics", () => {
+    test("200: should respond with 200 status code", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+    })
+    test("200: should respond with an array", () => {
+        return request(app)
+        .get('/api/topics')
+        .then(({body}) => {expect(body.topics).toBeInstanceOf(Array);
+        })
+    })
+    test("200: each topic should have a slug and a description", () => {
+        return request(app)
+        .get('/api/topics')
+        .then(({body}) => {
+            body.topics.forEach((topic) => {
+                expect(topic).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+            })
+        })
+    })
+})
```

Comment:
These 3 tests are all good, but they could be condensed into one test. I’ll leave it up to yourself if you wish to refactor!

Reply:
Hi, I separated them on David's suggestion! I'll leave them as they are for now as it helps my understanding but noted :)